### PR TITLE
Fix bunch of typos

### PR DESCRIPTION
**Pull request by surajssd**

### CODE REVIEW

File: deploy/helm/charts/crds/zfssnapshot.yaml [hunk 1]
```diff
@@ -194,7 +194,7 @@ spec:
             - volumeType
             type: object
           status:
-            description: SnapStatus string that reflects if the snapshot was cretaed
```
Comment (pawanpraka1) [inline deploy/helm/charts/crds/zfssnapshot.yaml:197]: we should bummp the chart version also.

Comment (surajssd) [reply deploy/helm/charts/crds/zfssnapshot.yaml:197]: done

**Review by shubham14bajpai — APPROVED** [review]
LGTM

**Review by pawanpraka1 — APPROVED** [review]
looks good. Thanks for this PR @surajssd.

**Pull request merged**